src/libs/actions/Task.js

```diff
@@ -624,11 +625,7 @@ function cancelTask(taskReportID, taskTitle, originalStateNum, originalStatusNum
         },
     ];
 
-    API.write('CancelTask', {taskReportID, optimisticReportActionID}, {optimisticData, failureData});
-}
-
-function isTaskCanceled(taskReport) {
-    return taskReport.stateNum === CONST.REPORT.STATE_NUM.SUBMITTED && taskReport.statusNum === CONST.REPORT.STATUS.CLOSED;
+    API.write('CancelTask', {, cancelledTaskReportActionID: optimisticReportActionID, taskReportID}, {optimisticData, failureData});
```

Comment:
Shouldn't have the comma here

Reply:
I'm so confused, that comma isn't there for me locally 😕